commands/bitswap.go

```diff
@@ -0,0 +1,32 @@
+package commands
+
+import (
+	"gx/ipfs/QmXixGGfd98hN2dA5YiPHWANY3sjmHfZBQk3mLiQUo6NLJ/go-bitswap"
+	"gx/ipfs/Qmde5VP1qUkyQXKCfmEUA7bP64V2HAptbJ7phuPp7jXWwg/go-ipfs-cmdkit"
+	"gx/ipfs/Qmf46mr235gtyxizkKUkTH5fo62Thza2zwXR4DWC7rkoqF/go-ipfs-cmds"
+)
+
+var bitswapCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Explore libp2p bitswap.",
+	},
+
+	Subcommands: map[string]*cmds.Command{
+		"stats": statsBitswapCmd,
+	},
+}
+
+var statsBitswapCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Show bitswap statistics.",
+	},
+
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		stats, err := GetPorcelainAPI(env).BitswapGetStats(req.Context)
+		if err != nil {
+			return err
+		}
+		return res.Emit(stats)
+	},
+	Type: &bitswap.Stat{},
```

Comment:
NON-blocking: FYI to @anorth @mishmosh , I thought about requesting a pretty printed text encoder for this command, but piping the existing json output to `jq` makes this very readable.